doc/unordered/benchmarks.adoc

```diff
@@ -278,13 +278,14 @@ max load factor 5
 
 |===
 
-== boost::unordered_flat_map
+== boost::unordered_(flat|node)_map
 
 All benchmarks were created using:
 
 * `https://abseil.io/docs/cpp/guides/container[absl::flat_hash_map^]<uint64_t, uint64_t>`
-* `boost::unordered_flat_map<uint64_t, uint64_t>`
 * `boost::unordered_map<uint64_t, uint64_t>`
```

Comment:
Should we replace `boost::unordered_map` with `absl::node_hash_map` for our benchmarks here?